### PR TITLE
Fix casing of switch [u->U]seSocketsHttpHandler

### LINKED ISSUE #1555: Poor documentation for System.Net.Http.useSocketsHttpHandler switch
At the ASP.NET Core remarks section:
* The switch may have a typo: System.Net.Http.__u__seSocketsHttpHandler (the 'u' is lowercase, however it is uppercase on other documentation pages).
* Values and Description seem to be mixed and/or missing.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 5c268ee4-aa76-5875-7162-58f02dc1c1a9
* Version Independent ID: c321117d-e7ec-21b1-0985-90a39191442d
* Content: [AppContext.SetSwitch(String, Boolean) Method (System)](https://docs.microsoft.com/en-us/dotnet/api/system.appcontext.setswitch?view=netframework-4.7.2#appcontextsetswitch-and-net-core)
* Content Source: [xml/System/AppContext.xml](https://github.com/dotnet/dotnet-api-docs/blob/master/xml/System/AppContext.xml)
* Product: **dotnet-api**
* GitHub Login: @dotnet-bot
* Microsoft Alias: **dotnetcontent**

### PR DESCRIPTION
**Pull request by am11**
* Few instances of `useSocketsHttpHandler` were missed in #2099.
* Also removed trailing whitespaces from the files this PR touched.
  * Used `nano` (command line) editor to open the file, `Ctrl+\` to enter search mode, `Alt+R` to enter regex mode, entered `[[:space:]]*$` as search key (to replace) and applied for all occurrences.
  * We can also do it in batch for all files by running the following command in Unix shell: `find . -type f -print0 | xargs -0 sed -i 's/[[:space:]]*$//'`

Fix #1555

### CODE REVIEW

**Comment by am11** [comment]
To review without whitespace changes, https://github.com/dotnet/dotnet-api-docs/pull/2337/files?w=1 :)

File: xml/System.Net.Http/SocketsHttpHandler.xml [hunk 1]
```diff
@@ -31,23 +31,23 @@ If this change is undesirable, you can configure your application to use the old
...
     "configProperties": {
-        "System.Net.Http.useSocketsHttpHandler": false
+        "System.Net.Http.UseSocketsHttpHandler": false
     }
   }
   ```
-  
-- By defining an environment variable named `DOTNET_SYSTEM_NET_HTTP_USESOCKETSHTTPHANDLER` and setting it to either `false` or 0.
+
+- By defining an environment variable named `DOTNET_SYSTEM_NET_HTTP_UseSocketsHttpHandler` and setting it to either `false` or 0.
```
Comment (mairaw) [inline xml/System.Net.Http/SocketsHttpHandler.xml]: was this change intentional?

Comment (am11) [reply xml/System.Net.Http/SocketsHttpHandler.xml]: Nope, I have reverted it. Thanks for the catch. :)

Comment (am11) [reply xml/System.Net.Http/SocketsHttpHandler.xml]: @mairaw, can this be complete? Thanks! :)
